homeassistant/components/anova/sensor.py

```diff
@@ -49,11 +52,15 @@ class AnovaSensorEntityDescription(
     AnovaSensorEntityDescription(
         key="state",
         translation_key="state",
+        device_class=SensorDeviceClass.ENUM,
+        options=[state.name for state in AnovaState],
```

Comment:
What should we do in the case of "no_state"?

Reply:
That should be fine I think. "no_state" should get turned into "No state" through translations. It's why I use name instead of value.

I create the stateby doing AnovaState(response_from_api) - when state is "" - it sets state = AnovaState.no_state

Then the value is saved to AnovaUpdateSensor as  the state.name